test/constructors.jl

```diff
@@ -65,5 +65,16 @@ module TestConstructors
     @test isequal(NullableArray{Bool, 2}(),
                   NullableArray{Bool, 2}(Array(Bool, 0, 0)))
 
-
+    # test conversion from arrays of nullables
+    array_of_nulls = Nullable{Int}[Nullable(1), Nullable(2), Nullable(3), Nullable()]
+    @test isa(NullableArray(array_of_nulls), NullableArray{Int,1})
+    @test isequal(NullableArray(array_of_nulls), NullableArray{Int,1}([1,2,3,4],[false,false,false,true]))
```

Comment:
Good tip, but `is`/`===`/`≡` for mutables checks if the memory addresses are the same, which isn't true here (my constructors copy the data so they won't end up with the same references).

For example:
```
julia> NullableArray([1,2,3]) === NullableArray([1,2,3])
false
```